contrib/bentoml/README.md

```diff
@@ -1,193 +1,209 @@
+        ]
+    )
 
-- Serve and test the service.
-    
+    # Return fraud if at least one model returns fraud
+    return any(results)
 ```
-!bentoml serve service.py:svc --reload
 
-2022-11-07T06:50:53+0000 [INFO] [cli] Prometheus metrics for HTTP BentoServer from "service.py:svc" can be accessed at <http://localhost:3000/metrics>.
-2022-11-07T06:50:53+0000 [INFO] [cli] Starting development HTTP BentoServer from "service.py:svc" listening on <http://0.0.0.0:3000> (Press CTRL+C to quit)
+## Deploy to Kubernetes Cluster
 
-```
-    
-![image](https://user-images.githubusercontent.com/861225/212856978-c8a24c4b-bc5b-4706-887e-81f5be914938.png)
+BentoML offers three custom resource definitions (CRDs) in the Kubernetes cluster.
 
-- Build bento
-    
-```
-!bentoml build
+- [BentoRequest](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bentorequest_crd.html) - Describes the metadata needed for building the container image of the Bento, such as the download URL. Created by the user.
+- [Bento](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bento_crd.html) - Describes the metadata for the Bento such as the address of the image and the runners. Created by users or by the `yatai-image-builder` operator for reconsiliating `BentoRequest` resources.
+- [BentoDeployment](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bentodeployment_crd.html) - Describes the metadata of the deployment such as resources and autoscaling behaviors. Reconciled by the `yatai-deployment` operator to create Kubernetes deployments of API Servers and Runners.
 
-Building BentoML service "credit_application:wly5lqc6ncpzwcvj" from build context "."
-Successfully built Bento(tag="credit_application:wly5lqc6ncpzwcvj").
-```
+![image](https://user-images.githubusercontent.com/861225/212857708-f96c9877-bb89-4afa-930a-1d2cb0300520.png)
 
-- Export bento to blob storage.
+Next, we will demonstrate two ways of deployment.
 
-```
-!bentoml export credit_application:wly5lqc6ncpzwcvj s3://your_bento_bucket/credit_application.wly5lqc6ncpzwcvj.bento
-```
+1. Deploying using a `BentoRequest` resource by providing a Bento
+2. Deploying Using a `Bento` resource by providing a pre-built container image from a Bento
 
-### Deploying to Kubernetes
+Users can choose to interact with Kubernetes through either `kubectl` or Kubernetes [Python client](https://github.com/kubernetes-client/python). The commands below demonstrates the depoyment using `kubectl`.
 
-![image](https://user-images.githubusercontent.com/861225/212857708-f96c9877-bb89-4afa-930a-1d2cb0300520.png)
+### Deploy with BentoRequest CRD
 
-Users can deploy bentos to the K8s cluster in one of the three ways.
+In this workflow, we will export the Bento to a remote S3 storage. After that, we will use the `yatai-image-builder` operator to containerize the Bento and push the resulting OCI image to a remote registry. Finally, we will use the `yatai-deployment` operator to deploy the containerized Bento image.
```

Comment:
Added a warning.

> WARNING: The `yatai-image-builder` operator requires root privileges because it needs to access the Docker daemon, which requires elevated permissions. Granting root privileges can potentially be dangerous, as it can give a user unrestricted access to the underlying operating system.